normits_demand/models/notem/production_models.py

```diff
@@ -329,26 +323,21 @@ def run(self,
             #  Output some audits of what demand was before and after control
             #  By segment.
             if self.constraint_paths is not None:
+                self._logger.info("No code implemented to constrain productions." )
```

Comment:
This should also be a debug call, handled in the same way as warnings

Reply:
Updated